README.md

```diff
@@ -1,7 +1,29 @@
-# Object-Service
+# object-service
```

Comment:
Why did you change the capital letters of the object-service , but didn't with installation and Requirements?